cumulus/client/consensus/aura/src/collators/slot_based/block_builder_task.rs

```diff
@@ -387,3 +433,64 @@ where
 		}
 	}
 }
+
+/// Translate the slot of the relay parent to the slot of the parachain.
+fn adjust_para_to_relay_parent_slot(
+	relay_header: &RelayHeader,
+	relay_chain_slot_duration: Duration,
+	para_slot_duration: SlotDuration,
+) -> Option<SlotInfo> {
+	let relay_slot = sc_consensus_babe::find_pre_digest::<RelayBlock>(&relay_header)
+		.map(|babe_pre_digest| babe_pre_digest.slot())
+		.ok()?;
+	let new_slot = Slot::from_timestamp(
+		relay_slot
+			.timestamp(SlotDuration::from_millis(relay_chain_slot_duration.as_millis() as u64))
+			.unwrap(),
```

Comment:
```suggestion
			.timestamp(SlotDuration::from_millis(relay_chain_slot_duration.as_millis() as u64))?,
```

Wild unwrap detected